### PR TITLE
Update hosting-a-web-page-on-your-pod

### PR DESCRIPTION
**Pull request by johanbove**
Added explanation on how to create a folder withing ``public`` and added the "Next Steps" plan as to how to get images hosted on the Pod.

### CODE REVIEW

**Comment by melvincarvalho** [comment]
Thanks for this, going to look at it in the next days.  Meanwhile I added a short chapter.

**Comment by johanbove** [comment]
I see. I resolved the conflict and merged with your latest changes.

**Comment by melvincarvalho** [comment]
@johanbove thank you!  Sorry didnt mean to give you more work to do.  I'll look at this.

File: hosting-a-web-page-on-your-pod.md [hunk 1]
```diff
@@ -2,15 +2,20 @@
 
 ### Introduction
 
-This hack enables you to host traditional web pages on your Solid pod.  Solid is 100% backwards compatible with the existing web.  Hosting a webpage on your pod couldn't be easier.  It can either be a normal HTML page.  Or interactive using JavaScript.
+This hack enables you to host traditional web pages on your Solid pod.  Solid is 100% backwards compatible with the existing web.  Hosting a webpage on your pod is free and couldn't be easier.  It can either be a normal HTML page.  Or interactive using JavaScript.
+
+We're going to create public web page hosted on your Solid pod, which can be accessed from:
+
+    https://{yourname.solidpod}/public/mysite/webpage.html
```
Comment (melvincarvalho) [inline hosting-a-web-page-on-your-pod.md]: I think I've done this first part!

Comment (johanbove) [reply hosting-a-web-page-on-your-pod.md]: Not sure I understand what you mean with your comment here. Can you please elaborate?

Comment (melvincarvalho) [reply hosting-a-web-page-on-your-pod.md]: Oh sorry, I was adding the text in parts.  I did that first part.  But now I think I've done all.

**Comment by melvincarvalho** [comment]
Given you write permission to the book (I think)

**Pull request closed without merging**